roles/consul/defaults/main.yml

```diff
@@ -29,9 +29,9 @@ consul_os: "\
   {% else %}\
     {{ ansible_system | lower }}\
   {% endif %}"
-consul_pkg: "consul{% if consul_enterprise %}-enterprise{% else %}{%endif%}_{{ consul_version }}_{{ consul_os }}_{{ consul_architecture }}.zip"
+consul_pkg: "consul{% if consul_enterprise %}-enterprise{% endif %}_{{ consul_version }}_{{ consul_os }}_{{ consul_architecture }}.zip"
```

Comment:
it looks like the "else" has been removed here. Why?

![image](https://github.com/vitabaks/postgresql_cluster/assets/37010174/659238be-8cb7-4519-9ce0-16b0821de29e)

Reply:
The {% else %} part was doing nothing, so it is unnecessary.

We can see that if the `consul_enterprise` condition is met, it adds "-enterprise," and that's all.

Therefore, if the condition is fulfilled, we will have:
consul-enterprise_{{ consul_version }}

If it is not fulfilled, we will have:
consul_{{ consul_version }}

In my opinion, this refactoring is justified

Reply:
ok, thanks